DynamicProgramming/LongestPalindromicSubsequence.java

```diff
@@ -0,0 +1,59 @@
+package DynamicProgramming;
+import java.lang.*; 
+import java.io.*; 
+import java.util.*; 
+  
+/**
+ * @author Matteo Messmer https://github.com/matteomessmer
+ * Algorithm explanation https://www.educative.io/edpresso/longest-palindromic-subsequence-algorithm
+ */
+public class LongestPalindromicSubsequence {
+    public static void main(String[] args) {
+        String a = "BBABCBCAB";
+        String b = "BABCBAB";
+
+        String aLPS = LPS(a);
+        String bLPS = LPS(b);
+
+        System.out.println(a + " => " + aLPS);
+        System.out.println(b + " => " + bLPS);
+    }
+
+    private static String LPS(String original) {
```

Comment:
Add throws IllegalArguementException clause to avoid runtime NullPointerException error